.travis.yml

```diff
@@ -2,11 +2,12 @@ language: go
 dist: xenial
 go:
   - 1.12.x
+  - 1.13.x
 script:
   - diff -u <(echo -n) <(gofmt -d *.go)
   - diff -u <(echo -n) <(golint $(go list -e ./...) | grep -v YAMLToJSON)
   - GO111MODULE=on go vet .
   - GO111MODULE=on go test -v -race ./...
   - git diff --exit-code
 install:
-  - go get golang.org/x/lint/golint
+  - GO111MODULE=off go get golang.org/x/lint/golint
```

Comment:
Alternatively, I think running this from a different directory would also work (that way it could still use go modules, but wouldn't modify the `go.mod`)

Reply:
Ack if we need to revisit this, will create a temp directory for installing stuff.